.github/workflows/ci.yml

```diff
@@ -221,7 +221,8 @@ jobs:
           token: ${{ secrets.CODECOV_TOKEN }}
 
   e2e-tests:
-    runs-on: ubuntu-22.04
+    # Use the CNCF's larger runners for E2Es, which otherwise take 20+ minutes
+    runs-on: oracle-16cpu-64gb-x86-64
```

Comment:
out of curiosity, how did you land on this runner? i think other repos that need larger runners, i.e. family providers, have mostly been using `ubuntu-latest-16-cores`:
https://github.com/crossplane-contrib/provider-upjet-aws/blob/main/.github/workflows/publish-provider-packages.yaml#L38

Reply:
https://cloud-native.slack.com/archives/C08P4HUFQ6M/p1745470313829149